hack/build/run-build.sh

```diff
@@ -3,7 +3,7 @@
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 # Default values
-default_git_repo_url=https://github.com/devfile-samples/devfile-sample-go-basic
```

Comment:
Since the sample devfile is no more available on the repo I get the below error:

```
STEP-ANNOTATE-TASK

acs-image-scan docker-build-rhtap-acs-deploy-check
taskrun.tekton.dev/docker-build-rhtap-acs-deploy-check annotated
taskrun.tekton.dev/docker-build-rhtap-acs-deploy-check annotated
taskrun.tekton.dev/docker-build-rhtap-acs-deploy-check annotated
taskrun.tekton.dev/docker-build-rhtap-acs-deploy-check annotated
STEP-ROX-DEPLOY-CHECK

Using rox central endpoint <endpoint-api>
Using gitops repository: https://github.com/devfile-samples/devfile-sample-go-basic-gitops
Cloning into 'gitops'...
fatal: could not read Username for 'https://github.com/': No such device or address
/tekton/scripts/script-1-tkxjt: line 27: cd: gitops: No such file or directory
List of files in gitops repository root:
total 0
drwxrwsrwx. 2 root 1000760000  6 Mar 14 06:45 .
drwxrwsrwx. 3 root 1000760000 24 Mar 14 06:45 ..
List of components in the gitops repository:
ls: cannot access 'components/': No such file or directory
Download roxctl cli
Performing scan for devfile-sample-go-basic component
Failed to find file to check: components/devfile-sample-go-basic/base/deployment.yaml
STEP-REPORT
```
So I am using https://github.com/prietyc123-qe-org/rhtap-go-sample